borg/selftest.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import time
+from unittest import TestResult, TestSuite, defaultTestLoader
```

Comment:
true. CryptoTestCase port to pytest (using pytest fixtures even) is in #912, which would take that off the list (emulating pytest is not an option; needing it at runtime is meh). The others wouldn't really profit from pytest's fixtures, they don't need that. I'd say:

* scratch CryptoTestCase from the list (with #912, not here)
* POST test cases stay with unittest for now